lib/rubill/base.rb

```diff
@@ -49,7 +49,13 @@ def self.delete(id)
     end
 
     def self.where(filters=[])
-      raise ArgumentError unless filters.all? { |f| f.is_a?(Query::Filter) }
+      raise ArgumentError unless filters.is_a?(Hash) || filters.all? { |f| f.is_a?(Query::Filter) }
```

Comment:
This will raise an error if filters is not a hash but is also not an array. What do you think about putting the `if filters.is_a?(Hash)...` logic before this raise statement, then `raise unless filters.is_an?(Array) && filters.all? ...`